scripts/west_commands/runners/intel_adsp.py

```diff
@@ -124,4 +125,8 @@ def flash(self, **kwargs):
         # not output to stdout directly. That means we can
         # make the log output to the PTY.
         if not self.pty:
-            self.check_call(self.log_cmd)
+            # Though it's encouraged to do this way due to
```

Comment:
I'm confused, commit message says something is _not_ encouraged, here it is. I guess both messages could be clearer.